torch/package/package_exporter.py

```diff
@@ -229,10 +229,16 @@ def __init__(
         torch._C._log_api_usage_once("torch.package.PackageExporter")
         self.debug = debug
         if isinstance(f, (Path, str)):
-            f = str(f)
+            import os
```

Comment:
Can you please place it at the top of the file? No reason to import it here

Reply:
sure